ext/AvizExt/viz/sensitivity.jl

```diff
@@ -437,18 +489,107 @@ function ADRIA.viz.outcome_map!(
 end
 function ADRIA.viz.outcome_map(
     rs::ResultSet,
-    si::YAXArray,
+    outcomes::Dataset,
     factors::Vector{Symbol};
     opts::OPT_TYPE=DEFAULT_OPT_TYPE(),
     fig_opts::OPT_TYPE=DEFAULT_OPT_TYPE(),
     axis_opts::OPT_TYPE=DEFAULT_OPT_TYPE()
 )
     f = Figure(; fig_opts...)
     g = f[1, 1] = GridLayout()
-    ADRIA.viz.outcome_map!(g, rs, si, factors; opts=opts, axis_opts=axis_opts)
+    ADRIA.viz.outcome_map!(g, rs, outcomes, factors; opts=opts, axis_opts=axis_opts)
+
+    return f
+end
+function ADRIA.viz.outcome_map!(
+    ax::Axis,
+    outcomes::YAXArray,
+    ms_factor::DataFrame,
+    f_vals::Vector{Float64};
+    opts::Dict=Dict(),
+    axis_opts::Dict=Dict()
+)
+    f_name = ms_factor.fieldname[1]
+    f_type::String = ms_factor.ptype[1]
+
+    if _is_discrete_factor(f_type)
+        # If categorical/discrete get categorical quantile
+        fv_s = _get_cat_quantile(
+            ms_factor, f_name,
+            collect(outcomes.axes[1])
+        )
+
+    else
+        # Otherwise use regular quantile
+        fv_s = round.(quantile(f_vals, collect(outcomes.axes[1])), digits=2)
+    end
+
+    if .!all(outcomes[CI=At("mean")] .== 0.0)
+        band!(
+            ax,
+            fv_s[.!ismissing.(outcomes[CI=At("lower")])],
+            collect(skipmissing(outcomes[CI=At("lower")])),
+            collect(skipmissing(outcomes[CI=At("upper")]))
+        )
+        scatterlines!(ax, fv_s, collect(outcomes[CI=At("mean")]); markersize=15)
+
+        if f_name == :guided
+            fv_labels = _get_guided_labels()
+            ax.xticks = (fv_s, fv_labels)
+            ax.xticklabelrotation = pi / 4
+        end
+    end
+    return ax
+end
+function ADRIA.viz.outcome_map(
+    rs::ResultSet,
+    outcomes::YAXArray,
+    factor::Symbol;
+    opts::Dict=Dict(),
+    fig_opts::Dict=Dict(),
+    axis_opts::Dict=Dict()
```

Comment:
This set of changes is replacing the new approach to defining default options for visualizations (see #777).

I'm assuming the change is being accidentally overwritten.